src/shared/utilities/pathUtils.ts

```diff
@@ -20,8 +20,18 @@ function isUncPath(path: string) {
  * @param logicalRoot Resolve relative paths against this directory
  * @param path1 Path to compare
  * @param path2 Path to compare
+ * @param stripLeadingVar Optionally strip leading VS Code launch config variables
  */
-export function areEqual(logicalRoot: string | undefined, path1: string, path2: string): boolean {
+export function areEqual(
+    logicalRoot: string | undefined,
+    path1: string,
+    path2: string,
+    stripLeadingVar?: boolean
+): boolean {
+    if (stripLeadingVar) {
+        path1 = path1.replace(/^\${[a-zA-Z]{1,}}\//, '')
+        path2 = path2.replace(/^\${[a-zA-Z]{1,}}\//, '')
```

Comment:
areEqual() does not require `/` path separators until now. Should this be:
```suggestion
        path2 = path2.replace(/^\${[a-zA-Z]{1,}}[\/\\]/, '')
```

Reply:
That's fair, I was mainly accounting for the fact that the current impl specifically adds `/`. Might need to be `\\\\` for Windows?